extensions/funqy/funqy-http/runtime/src/main/java/io/quarkus/funqy/runtime/bindings/http/VertxRequestHandler.java

```diff
@@ -106,9 +109,8 @@ public void handle(RoutingContext request) {
     private void dispatch(RoutingContext routingContext, FunctionInvoker invoker, Object input) {
         ManagedContext requestContext = beanContainer.requestContext();
         requestContext.activate();
-        QuarkusHttpUser user = (QuarkusHttpUser) routingContext.user();
-        if (user != null && association != null) {
-            association.setIdentity(user.getSecurityIdentity());
+        if (association != null) {
+            ((Consumer<Uni<SecurityIdentity>>) association).accept(QuarkusHttpUser.getSecurityIdentity(routingContext, null));
```

Comment:
Note that this is because I need to add a new method to an interface in quarkus-security. If this PR is otherwise fine I will add it and cut a release.